providers/openebs/installers/operator/0.6/ansible/openebs_cleanup.yaml

```diff
@@ -0,0 +1,31 @@
+# TODO 
+# Delete openebs operator
+
+- hosts: localhost
+  connection: local
+
+  vars_files:
+  - vars.yaml
+
+  tasks:
+    - block:
+          - name: Cleaning openebs operator
+            shell: "{{ kubeapply }} delete -f {{ openebs_operator_link }}" 
+            args:
+              executable: /bin/bash
+            ignore_errors: True
+
+          - name: Confirm pods has been deleted
+            shell: kubectl get pods --all-namespaces 
+            args:
+              executable: /bin/bash
+            register: result
+            until: '"maya-apiserver" and "openebs-provisioner" and "openebs-snapshot-operator" not in result.stdout'

```

Comment:
can we convert this into a list (with_items) preferably that allows us to scale the number of items to check going ahead ?

Reply:
sure nice enhancement